### DISCUSSION TITLE
How can i host / deploy saleor to railway.app?

### DISCUSSION THREAD

**Discussion by MoathCodes** [saleor/saleor#11459] [Q&A] [answered]
Hi,
I'm trying to deploy saleor to railway with the Graphql playground but i can't seem to figure it out.
if anyone can direct me to clear instructions I would greatly appreciate it.
Thank you.

**Comment by karolkielecki** [comment, marked as answer]
Hi @IronDragonDev, Unfortunately, I'm not familiar with railway.app. Have you tried our Saleor Cloud? You can create a [free developer account](https://cloud.saleor.io/signup).

**Reply by MoathCodes** [reply]
Thank you, I wasn't aware that Saleor Cloud had a free plan.
It will help me a lot developing my application.
